auth-web/src/views/auth/home/MaintainBusinessView.vue

```diff
@@ -36,35 +35,50 @@
           <learn-more-button
            :redirect-url="learnMoreUrl"
           />
-          <p v-if="!userProfile" class="mt-5">Don't have your Cooperative Passcode?
+          <p v-if="!userProfile" class="mt-5">
+            Don't have your Cooperative Passcode?
             <v-menu top offset-y v-model="contactUsPopover" :close-on-content-click="false" attach="#maintain-info-container">
               <template v-slot:activator="{ on, attrs }">
                 <span v-bind="attrs" v-on="on" class="popover-activator">Contact Us</span>
               </template>
               <v-card class="contact-popover">
                 <v-list class="pa-5" max-width="30rem">
                   <v-list-item class="pb-3">
-                    <v-list-item-title class="popover-title mt-5 font-weight-bold">Don't have a Passcode for your Cooperative<br>Association?</v-list-item-title>
+                    <v-list-item-title class="popover-title mt-5 font-weight-bold">
+                      Don't have a Passcode for your Cooperative<br>Association?
```

Comment:
This is possibly the wrong verbiage, here and below. See https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/6888#issuecomment-815237603 and responses.